test/integration/post.test.js

```diff
@@ -0,0 +1,56 @@
+const { expect } = require('chai');
+const {
+  startServer,
+  stopServer,
+  startBrowser,
+  stopBrowser,
+  TEST_URL,
+} = require('./utils');
+
+describe('Post', () => {
+  let page;
+
+  before(async function before() {
+    this.timeout(120000);
+    await startServer();
+  });
+
+  beforeEach(async () => {
+    page = await startBrowser();
+  });
+
+  afterEach(async () => {
+    page = await stopBrowser();
+  });
+
+  after(() => stopServer());
+
+  it('Renders the expected interlinks', async () => {
+    const testPages = [
+      {
+        path: '/en/publish/2020/12/17/future-of-government-modernizing-hr-teams-in-watershed-moment.html',
+        links: ['e-signatures'],
+      },
+      {
+        path: '/en/publish/2020/12/18/annual-end-of-year-predictions-cx-best-practices.html',
+        links: ['customer intelligence', 'customer profiles', 'customer experience', 'customer data', 'customer journey'],
+      },
+      {
+        path: '/en/publish/2020/10/23/optimize-email-deliverability-2020-holiday-season.html',
+        links: [],
+      }
+    ]
```

Comment:
I assume this is directly hitting production articles together with the interlink production config... 
If authors change the content or the keywords to link, the tests get broken... We should test on content we have control on.

Reply:
You're right, I'll record them.